src/commands/quiz.ts

```diff
@@ -0,0 +1,91 @@
+import { Command } from '../types';
+import fetch from 'node-fetch';
+
+const MAX_QUESTIONS = 10;
+const USAGE_MESSAGE = `Format: !quiz język poziom(opcjonalny) ilość(opcjonalny).
+Dostępne wartości:
+* język: html, css, js, angular, react, git, other
+* poziom: junior, mid, senior
+* ilość: [1 - ${MAX_QUESTIONS}] - ile pytań wylosować
+`;
+
+const Levels = ['junior', 'mid', 'senior'];
```

Comment:
Jak są to stałe, to LEVELS
```suggestion
const LEVELS = ['junior', 'mid', 'senior'];
```